tests/std/tests/GH_004201_chrono_formatter/test.cpp

```diff
@@ -0,0 +1,15 @@
+#include <cassert>
+#include <chrono>
+#include <format>
+#include <iostream>
+#include <string>
+#include <thread>

```

Comment:
`<iostream>` and `<thread>` were used in #4201, but not in this test, so they should be dropped here.